openedx/core/djangoapps/content/course_overviews/models.py

```diff
@@ -0,0 +1,251 @@
+"""
+Declaration of CourseOverview model
+"""
+
+from datetime import datetime
+import json
+
+import django.db.models
+from django.db.models.fields import BooleanField, DateTimeField, DecimalField, TextField
+from django.utils.timezone import UTC
+from django.utils.translation import ugettext
+
+from lms.djangoapps.certificates.api import get_active_web_certificate
+from lms.djangoapps.courseware.courses import course_image_url
+from util.date_utils import strftime_localized
+from xmodule import course_metadata_utils
+from xmodule.modulestore.django import modulestore
+from xmodule_django.models import CourseKeyField, UsageKeyField
+
+
+class CourseOverview(django.db.models.Model):
+    """
+    Model for storing and caching basic information about a course.
+
+    This model contains basic course metadata such as an ID, display name,
+    image URL, and any other information that would be necessary to display
+    a course as part of a user dashboard or enrollment API.
+    """
+
+    # Course identification
+    id = CourseKeyField(db_index=True, primary_key=True, max_length=255)  # pylint: disable=invalid-name
+    _location = UsageKeyField(max_length=255)
+    display_name = TextField(null=True)
+    display_number_with_default = TextField()
+    display_org_with_default = TextField()
+
+    # Start/end dates
+    start = DateTimeField(null=True)
+    end = DateTimeField(null=True)
+    advertised_start = TextField(null=True)
+
+    # URLs
+    course_image_url = TextField()
+    facebook_url = TextField(null=True)
+    social_sharing_url = TextField(null=True)
+    end_of_course_survey_url = TextField(null=True)
+
+    # Certification data
+    certificates_display_behavior = TextField(null=True)
+    certificates_show_before_end = BooleanField()
+    has_active_web_certificates = BooleanField()
```

Comment:
rename to either `has_any_active_web_certificate` or `has_active_web_certificate` (without the ending `s`).